### ISSUE TITLE
`[no-expression-in-message]` doesn't allow JSX whitespace `{' '}`

### ISSUE THREAD

**Issue by dannydtk** [lingui/eslint-plugin#67]
## Current behavior

When you have a piece of JSX like this in order to preserve the whitespace between the text and the inline component:

```tsx
<Trans>
  Did you mean{' '}
  <button className="some-class" onClick={doSmth} type="button">
    {option}
  </button>
  ?
</Trans>
```

or

```tsx
<Trans>
  Did you mean {}
  <button className="some-class" onClick={doSmth} type="button">
    {option}
  </button>
  ?
</Trans>
```

then the `lingui/no-expression-in-message` rule will be triggered with the following message:

> Should be ${variable}, not ${object.property} or ${my_function()} 

## Expected behavior

The `lingui/no-expression-in-message` rule is not triggered for JSX whitespaces.

## Env
```json
{
  "eslint-plugin-lingui": "0.5.0",
  "eslint": "8.57.1"
}
```

**Comment by timofei-iatsenko** [comment]
Thanks for report. I'll take a look
